src/templates/static/platforms.jsx

```diff
@@ -121,25 +121,29 @@ const Platform = () => (
         <div className='fill-bg-color gr-padding-30 mobile-hide' data-show='-eucountry'>
             <div className='container gr-padding-30 gr-child'>
                 <h2 className='primary-color center-text'>
-                    {it.L('Deriv — an all-new trading experience')}
+                    {it.L('Introducing Deriv.com')}
                 </h2>
+                <p className='center-text gr-child'>{it.L('A rebranding of Binary.com to an all-new trading experience')}</p>
                 <div className='gr-row gr-row-align-around'>
                     <div className='gr-5 gr-12-p gr-12-m'>
                         <ul className='bullet b-m-md'>
                             <li>
-                                <strong>{it.L('Powerful and intuitive')}</strong>
-                                <br />
-                                {it.L('Easy to use and feature-rich.')}
+                                {it.L('A single, comprehensive trading service with ')}
+                                <strong>{it.L('4 advanced platforms')}</strong>
```

Comment:
you should combine this into one string and use placeholders for the strong tags. otherwise translators will not have context and will not be able to position the words in the sentence correctly for their grammar.

Reply:
done